src/components/src/utils/AuthService.ts

```diff
@@ -156,6 +156,7 @@ export function logout() {
       window.dispatchEvent(new CustomEvent('authHeaderChanged', { detail: { authHeader: null, file: 'AuthService.3' } }));
       localStorage.removeItem(`${Config.cortexApi.scope}_oAuthUserName`);
       localStorage.removeItem(`${Config.cortexApi.scope}_b2bCart`);
```

Comment:
We can remove more of these localStorage.remove calls since the openID items are no longer used. If we no longer set a value in our app, we should be able to remove the mention of it here, and in other files.